### PR TITLE
Core 10: Cannot create pk values with null and 0

### PR DESCRIPTION
**Pull request by cmelchior**

### CODE REVIEW

**Comment by jedelbo** [comment]
This will not compile before the next version of core is used.

File: tests/object.cpp [hunk 1]
```diff
@@ -923,6 +923,18 @@ TEST_CASE("object") {
         REQUIRE(obj.obj().get<String>(col_pk_str) == "value");
     }
 
+    SECTION("create null and 0 primary keys for Int types") {
```
Comment (cmelchior) [inline tests/object.cpp]: Shouldn't we add unit tests for all the primary key types?

**Comment by jedelbo** [comment]
Now waiting for new sync release

File: tests/object.cpp (hunk 1 above)
Comment (jedelbo) [reply tests/object.cpp]: Added a test with ObjectId as primary key. We have one for string.

**Pull request merged**